README.md

```diff
@@ -67,4 +92,4 @@ spec:
     server: localhost
   persistentVolumeReclaimPolicy: Recycle
 PV
```

Comment:
We'll have to expand upon the system requirements moving forward as 5 PV's will be required